sources/packages/backend/libs/integrations/src/sfas-integration/sfas-files/sfas-individual-record.ts

```diff
@@ -181,4 +181,64 @@ export class SFASIndividualRecord extends SFASRecordIdentification {
   get partTimeMSFAAEffectiveDate(): Date | null {
     return parseDate(this.line.substring(241, 249));
   }
+  /**
+   * Initials of applicant (individual_alias_current_view.initials).
+   */
+  get initials(): string {
+    return this.line.substring(249, 252)?.trim();
+  }
+  /**
+   * Line 1 of the applicant's address (address_current_view_unique.address_1).
+   */
+  get addressLine1(): string {
+    return this.line.substring(252, 292)?.trim();
+  }
+  /**
+   * Line 2 of the applicant's address.
+   */
+  get addressLine2(): string {
+    return this.line.substring(292, 332)?.trim();
+  }
+  /**
+   * City name the applicant's address (address_current_view_unique.city).
+   */
+  get city(): string {
+    return this.line.substring(332, 357)?.trim();
+  }
+  /**
+   * Province or state code - only exists if in Canada or US (address_current_view_unique.prov_cde).
+   */
+  get provinceState(): string {
+    return this.line.substring(357, 361)?.trim();
+  }
+  /**
+   * Country code (address_current_view_unique.country_cde).
+   */
+  get country(): string {
+    return this.line.substring(361, 365)?.trim();
+  }
+  /**
+   * Phone number of the applicant (address_current_view_unique.phone_num).
+   */
+  get phoneNumber(): number {
+    return +this.line.substring(365, 381);
```

Comment:
For records without the phone number, will it save zero to DB instead of null?

Reply:
That is something missed. I will update.